### DISCUSSION TITLE
How to check if Item is purchased or not ?

### DISCUSSION THREAD

**Discussion by devnilspatel** [akshaaatt/Google-IAP#39] [Q&A]
Hi, team I got your library from anjlib comment sections. I am old user of that library. When I want to check if any Product id is already purchased or not i mostly use _`billingProcessor.listOwnedProducts()`_ and check it. In your library how can we check it ?

Thanks a lot.

**Comment by akshaaatt** [comment]
Please update to the latest implementation
